p2p/p2p.go

```diff
@@ -182,6 +184,34 @@ func multiAddrViaRelay(relayPeer Peer, peerID peer.ID) (ma.Multiaddr, error) {
 	return transportAddr.Encapsulate(relayAddr), nil
 }
 
+func NewEventCollecter(tcpNode host.Host) lifecycle.HookFuncCtx {
+	return func(ctx context.Context) {
+		sub, err := tcpNode.EventBus().Subscribe(new(event.EvtLocalReachabilityChanged))
+		if err != nil {
+			log.Error(ctx, "Subscribe libp2p events", err)
```

Comment:
Nit,  probably not worth doing if it's an error deep in prom logs and not user facing  code.  
```suggestion
			log.Error(ctx, "There was an error subscribing to libp2p LocalReachabilityChanged events", err)
```

Reply:
yeah, this is deep inside, should never happen in practise